_docs/quickstarts/forseti-security/index.md

```diff
@@ -39,63 +67,28 @@ file based on your input.
       cd forseti-security/scripts/gcp_setup
       ```
 
-  1. Start the setup process:
+  1. Invoke the setup:
   
       If you downloaded a certain release of Forseti, specify the release version
       to the setup wizard.
-
-      Setup wizard is not available prior to 1.1.0.
-      
-      _e.g. Deploy 1.1.0:_
-      
-      ```bash
-      python setup_forseti.py --version 1.1.0
-      ```
-  
-     _Default: runs master branch_
      
       ```bash
+      # Default: master branch
       python setup_forseti.py
-      ```
-      
-      _Run a certain branch:_
       
-      ```bash
+      # Specify a branch (e.g. develop)
       python setup_forseti.py --branch develop
+      
+      # Display help information
+      python setup_forseti.py -h
       ```
 
-  1. Follow the prompts to download and install the
-  [gcloud command-line tool](https://cloud.google.com/sdk/gcloud/).
-  1. Follow the prompts to authenticate your GCP account in a web browser,
-  then return to the command-line.
-  1. Enter the organization for which you want to deploy Forseti, if prompted.
-  1. Create a new project or enter an existing project ID you want to use for
-  Forseti Security. Note that it's best to use a project dedicated to running
-  Forseti.
-  1. The setup wizard checks for valid configurations or creates a new one,
-  then it checks if billing is enabled for your project.
-      * If billing isn't enabled, follow the prompts to enable billing, then
-    return to the command-line.
-  1. Next, the setup wizard automatically enables required APIs:
-  
-  {% include docs/required_apis.md %}
-  
-  1. Create a new service account or enter an existing service account for
-  accessing GCP.
-  1. Optionally create a new service account or enter an existing service
-  account for getting GSuite groups.
-  1. Next, the setup wizard automatically assigns the required roles to the GCP service
-  account. For details about the required roles, see [Forseti Security Best Practices Guide]({% link _docs/guides/best-practices.md %})
-  1. Enter a name for your bucket and Cloud SQL instance.
-  1. Setup wizard will create a Deployment Manager template based on your input
-  as well as ask whether you want to create the deployment.
-  1. Optionally create the bucket and a Cloud SQL instance. Only do this if you 
-  do not plan to create a deployment via Deployment Manager.
-  1. If you set up a service account to retrieve GSuite groups, follow the
-  post-setup instructions to enable your GSuite service account for domain-wide delegation.
-
-Forseti Security is now set up for your project and you can enable modules
-to use for your resources.
+  1. Setup will infer the necessary information to install Forseti. You will be 
```

Comment:
Is this already supposed to work? I followed the steps here and I'm getting the old prompts to authenticate, choose organization, setup the project, etc.